stable/ambassador/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.85.0
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 5.1.0
+version: 5.1.1
```

Comment:
this would be a breaking change and not a patch since its removing the podLabels, I'd prefer if we can make this a non breaking change for now.